demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -842,3 +845,25 @@ def is_valid_description(self, beta_integration: bool = False) -> bool:
             if not description_validator.is_valid():
                 return False
         return True
+
+    def is_there_a_runnable(self) -> bool:
+        """Verifies there's at least one runnable command.
+            at least one of:
+            command in commands section
+            isFetch
+            feed
+            long-running
+
+        Returns:
+            if there's at least one runnable in the yaml
+        """
+        script = self.current_file.get('script', {})
+
+        if not any([
+            script.get('commands'), script.get('isfetch', script.get('isFetch')), script.get("feed"), script.get('longRunning')]
+        ):
+            self.is_valid = False
+            print_error(f"{self.file_path}: Could not find any runnable command in the integration."
```

Comment:
do not use `print_error`, create a new error and error-code using the as seen in errors.py and call it via `handle_error()`